insights/parsers/sssd_logs.py

```diff
@@ -42,61 +45,45 @@ class SSSDLog(LogFileOutput):
         (Tue Feb 14 09:45:06 2015) [sssd] [monitor_hup] (0x0020): Received SIGHUP.
         (Tue Feb 14 09:45:07 2015) [sssd] [te_server_hup] (0x0020): Received SIGHUP. Rotating logfiles.
 
+    Each line is parsed into a dictionary with the following keys::
+
+        * **timestamp** - the date of the log line (as a string)
```

Comment:
@xiangce did you mean for this to be a bullet list with formatted terms?  Because of the `::` above the formatting chars are being treated as literals.

Reply:
Thank you @bfahr , I will uniform these docstrings in next commit.